dev/docker/tools/docker-connector.conf

```diff
@@ -6,3 +6,4 @@
 # Fixed docker container network subnet in Gravitino integration testing module
 # Generate command: `docker network ls --filter driver=bridge --format "{{.ID}}" | xargs docker network inspect --format "route {{range .IPAM.Config}}{{.Subnet}}{{end}}" > ${bin}/docker-connector.conf`
 route 10.20.30.0/28
+route 10.20.30.16/28
```

Comment:
Why do we need to change this route configuration?

Reply:
Docker Compose uses the 10.20.30.16/28 network. Because if you use the integrated testing network, manually starting Docker Compose will result in errors due to the lack of network.